tests/basic_one_way_send.go

```diff
@@ -0,0 +1,84 @@
+package tests
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ava-labs/teleporter/tests/utils"
+
+	"github.com/ava-labs/awm-relayer/messages/teleporter"
+	"github.com/ava-labs/subnet-evm/interfaces"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	. "github.com/onsi/gomega"
+)
+
+// Ginkgo describe node that acts as a container for the teleporter e2e tests. This test suite
+// will run through the following steps in order:
+// 1. Send a transaction to the Teleporter contract on Subnet A
+// 2. Aggregate signatures and send the Warp message to Subnet B
+// 3. Verify receipt of the message on Subnet B
+func BasicOneWaySend() {
+	var (
+		teleporterMessageID *big.Int
+	)
+
+	subnetAInfo := utils.GetSubnetATestInfo()
+	subnetBInfo := utils.GetSubnetBTestInfo()
+	teleporterContractAddress := utils.GetTeleporterContractAddress()
+	fundedAddress, fundedKey := utils.GetFundedAccountInfo()
+
+	//
+	// Send a transaction to Subnet A to issue a Warp Message from the Teleporter contract to Subnet B
+	//
+	ctx := context.Background()
+
+	sendCrossChainMessageInput := utils.SendCrossChainMessageInput{
+		DestinationChainID: subnetBInfo.BlockchainID,
+		DestinationAddress: fundedAddress,
+		FeeInfo: utils.FeeInfo{
+			ContractAddress: fundedAddress,
+			Amount:          big.NewInt(0),
+		},
+		RequiredGasLimit:        big.NewInt(1),
+		AllowedRelayerAddresses: []common.Address{},
+		Message:                 []byte{1, 2, 3, 4},
+	}
+	signedTx := utils.CreateSendCrossChainMessageTransaction(ctx, subnetAInfo, sendCrossChainMessageInput, fundedAddress, fundedKey, teleporterContractAddress)
+
+	log.Info("Sending Teleporter transaction on source chain", "destinationChainID", subnetBInfo.BlockchainID, "txHash", signedTx.Hash())
+	receipt := utils.SendTransactionAndWaitForAcceptance(ctx, subnetAInfo.ChainWSClient, signedTx)
+
+	sendCrossChainMessageLog := receipt.Logs[0]
+	var event utils.SendCrossChainMessageEvent
+	err := teleporter.EVMTeleporterContractABI.UnpackIntoInterface(&event, "SendCrossChainMessage", sendCrossChainMessageLog.Data)
+	Expect(err).Should(BeNil())
+
+	teleporterMessageID = event.Message.MessageID
```

Comment:
can we add extra verification about other event fields like `destinationChainID`, and comparing the even'ts teleporter message fields compared to `sendCrossChainMessageInput`?

Reply:
Added these checks, along with utilizing the abigen Go bindings.